app/controllers/users_controller.rb

```diff
@@ -1,5 +1,29 @@
 class UsersController < Clearance::UsersController
+  before_action :admin_user, only: [:destroy, :index]
+
   def show
     @user = current_user
   end
+
+  def index
+    @users = User.all.order(created_at: :desc).paginate(page: params[:page], per_page: 15)
+  end
+
+  def destroy
+    user = User.find(params[:id])
+    if user.admin?
+      redirect_to users_path
+      flash[:danger] = "Cannot delete admin user."
+    else
+      user.destroy
+      redirect_to users_path
+      flash[:success] = "User successfully deleted."
+    end
+  end
+
+  private
+
+  def admin_user
```

Comment:
This method name indicates that the method returns an admin user, which doesn't seem to be the case. 😅 How about `#require_admin_privileges!`